packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanel.tsx

```diff
@@ -672,7 +688,14 @@ export function TransferPanel() {
             `${selectedToken?.address} is ${description}; it will likely have unusual behavior when deployed as as standard token to Arbitrum. It is not recommended that you deploy it. (See https://developer.offchainlabs.com/docs/bridging_assets for more info.)`
           )
         }
-        if (isConnectedToArbitrum.current) {
+
+        const isParentChainEthereum = isNetwork(l1Network.id).isEthereum
```

Comment:
NIT: Maybe abstract this to `useParentChain` hook later to reduce duplication of logic.

Reply:
I can do those right after it goes live